### DISCUSSION TITLE
Distance info and EDDN integration

### DISCUSSION THREAD

**Discussion by ayaanrauf** [iaincollins/icarus#3] [Ideas & Suggestions]
Before I begin, I just wanted to say that this app is really cool and does help me a lot when I am using it.
EDDN integration - that way we won't have to run another app with the integration (EDMC, EDDiscovery, etc) to be able to help update market information.
Distance - when displaying a star system that is not the one that you are in, it shows the distance between you and that star
These are just ideas and they don't have to be added but I think they might help the app become better. What do you all think?

**Comment by iaincollins** [comment]
Thanks! Sorry I just noticed I'd left discussions open and didn't realize folks were posting here.

Agree, I'd love to do EDDN integration. I am currently using it alongside EDDI (which has a nice voice component too) for that reason. It's a bit of work, as it doesn't take raw journal messages but rather only accepts specific custom schemas - which is a shame - but is definitely on the list of things I want to do.

I agree, it would be good to display distance to a star that is being viewed remotely, to other stars in the current route and to highlight other nearby stars and their distances. I am still working on improving the Navigation UI but [displaying distances is not hard to do](https://www.reddit.com/r/EliteDangerous/comments/f3q6ya/how_can_i_find_the_distance_between_systems/) and something I should add in.

**Reply by ayaanrauf** [reply]
Okay, thank you!

**Comment by iaincollins** [comment]
Quick update to say the distance to systems in the System Map and Route views is now live in v0.4.1
https://github.com/iaincollins/icarus/releases/tag/v0.4.1

It also now displays expanded system information (controlling faction, system state, economy, security, allegiance, government) for all systems - local and remote - courtesy of the fantastic EDSM.

**Reply by ayaanrauf** [reply]
Thanks! Sorry, I didn't see this until now.
